# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/spellchecker/TextServicesDocument.h
# Commit: 3ba594b4b02d
# Full Hash: 3ba594b4b02d05812389723fc5bca2bc0894cb55
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-07-13 21:42:32
# Regressor Bug: 1718924
# File Overlap Count: 2
# Description:
#   Bug 1718924 - part 9: Rename `TextServicesDocument::mSel(Start|End)Offset` r=m_kato
#   
#   Now, the meaning of `OffsetEntry` is clear.  Therefore, this patch adds comment
#   explaining the class and its members.
#   
# ==============================================================================

diff -r fe76633b0f3b -r 3ba594b4b02d editor/spellchecker/TextServicesDocument.h
--- a/editor/spellchecker/TextServicesDocument.h	Tue Jul 13 09:31:34 2021 +0000
+++ b/editor/spellchecker/TextServicesDocument.h	Tue Jul 13 09:42:34 2021 +0000
@@ -64,8 +64,9 @@
   //       simpler.
   Maybe<size_t> mSelStartIndex;
   Maybe<size_t> mSelEndIndex;
-  Maybe<uint32_t> mSelStartOffset;
-  Maybe<uint32_t> mSelEndOffset;
+  // Selected start and end offset in all text in a block element.
+  Maybe<uint32_t> mSelectionStartOffsetInTextInBlock;
+  Maybe<uint32_t> mSelectionEndOffsetInTextInBlock;
 
   uint32_t mTxtSvcFilterType;
   IteratorStatus mIteratorStatus;
